lichess-bot.py

```diff
@@ -305,8 +307,9 @@ def log_proc_count(change, queued, used):
 
             if last_check_online_time.is_expired():
                 if not li.is_online(user_profile["id"]):
-                    logger.info("Will reset connection with lichess")
-                    li.reset_connection()
+                    logger.info("Will restart lichess-bot")
+                    restart = True
+                    terminated = True
```

Comment:
Instead of setting `terminated = True` here, change the while-loop to `while not terminated and not restart:`. In the `while restart:` loop below, if the users pushes `Ctrl-C` during the 10-second sleep, it will be ignored since `terminated` will be set back to `False` once the sleep ends.